public/js/formSubmit.js

```diff
@@ -1,158 +1,158 @@
-                        }
-                    } else {
-                        console.log('Can not write validation error ' + key);
-                    }
-                }
-            }
-        } else if (request.status >= 500 && request.status < 600) {
-            alert('Er is een serverfout opgetreden');
-        } else {
-            alert('Onbekende fout opgetreden in antwoord van de server');
-        }
-    };
-    request.onerror = function() {
-        alert('Onbekende fout opgetreden bij het maken van de verbinding');
-    };
-    request.send(data);
-};
-
-document.addEventListener('submit', function (event) {
-    // implementatie voor schuldeiser
-    if (window.schuldhulp._.matches(event.target, '.ajax[name="schuldeiser_form"]')) {
-        window.schuldhulp.formSubmit(event, null, function (response, data, dom, event) {
-            window.location.replace(window.location.href.split('#')[0]);
-        });
-    }
-    
-    // implementatie voor voorlegger
-    if (window.schuldhulp._.matches(event.target, '.ajax[name^="voorlegger_"]')) {
-        window.schuldhulp.formSubmit(event, null, function (response, data, dom, event) {
-            dom.parentNode.replaceChild(response[0], dom);
-        });
-        return;
-    }
-    
-    // implementatie voor schuld item
-    if (window.schuldhulp._.matches(event.target, '.ajax[name="schuld_item_form"]')) {
-        window.schuldhulp.formSubmit(event, function (data, dom, event) {
-            if (document.activeElement.classList.contains('remove')) {
-                data.append(document.activeElement.getAttribute('name'), document.activeElement.value);
-            }
-            return data;
-        }, function (response, data, dom, event) {
-            if (data.get('schuld_item_form[verwijderd]') == true) {
-                dom.parentNode.parentNode.parentNode.parentNode.removeChild(dom.parentNode.parentNode.parentNode);
-            } else {
-                // datepicker opnieuw toewijzen
-                elms = response[0].querySelectorAll('.rome');
-                for (var i = 0; i < elms.length; i ++) {
-                    window.schuldhulp._.datePicker(elms.item(i));
-                }
-                
-                var original = dom.parentNode.parentNode.parentNode.parentNode.querySelector('#' + response[0].getAttribute('id'));
-                if (original) {
-                    original.parentNode.replaceChild(response[0], original);
-                } else {
-                    dom.parentNode.parentNode.parentNode.querySelector('.accordion-container').appendChild(response[0]);
-                    dom.reset();
-                }
-            }
-        });
-        return;
-    }
-    
-    // algemene implementatie
-    if (window.schuldhulp._.matches(event.target, '.ajax')) {
-        window.schuldhulp.formSubmit(event, null, function () {});
-        return;
-    }
-});
+window.schuldhulp = window.schuldhulp || {};
+
```

Comment:
Hier zijn de codewijzigingen niet leesbaar. Als er geen codewijzigingen zijn, moet dit stuk er uit of anders alleen de diffs tonen.

Reply:
Zie mijn [algemene reactie](https://github.com/Amsterdam/schulddossier/pull/277#issuecomment-1097824167), er is gelukkig een mogelijkheid in GitHub aanwezig om deze whitespace change weg te filteren.